src/less/education-notice/education-notice.less

```diff
@@ -0,0 +1,69 @@
+@import "../variables/variables.less";
+@import "../mixins/private/token-mixins.less";
+
+.education-notice {
+    .background-color-token(education-notice-default-background-color, color-background-secondary);
+    .border-radius-token(education-notice-border-radius, border-radius-100);
+    color: var(--color-foreground-primary);
+    font-size: var(--font-size-default);
+    padding: var(--spacing-300);
+}
+
+.education-notice--prominent {
+    .background-color-token(education-notice-education-background-color, color-background-education);
+}
+
+.education-notice__header {
+    display: flex;
+}
+
+.education-notice__title {
+    flex-grow: 1;
+    font-size: var(--font-size-medium);
+    font-weight: bold;
+    line-height: 1.5em;
+    margin: 0;
+    margin-inline-start: var(--spacing-150);
+}
+
+.education-notice__main {
+    margin-top: var(--spacing-200);
+}
+
+.education-notice__dismiss {
+    margin-inline-start: var(--spacing-150);
+}
+
+.education-notice__header .icon--prominent {
+    color: var(--color-foreground-accent);
```

Comment:
Would this be used independent of `education-notice--prominent`? 
If not, can we instead use the `education-notice--prominent` modifier and apply it to all icons? 

Not sure if I like the name too, but not sure what we should change it to.

Reply:
The notice can be prominent and the icon can be prominent, and there is no logical synchronized conditions that we can be deterministic about the icon based on the notice or vice versa.

`prominent` was the term used by DS so I simply piggybacked on that instead of creating another name so the design docs would be inline with the implementation. That said, if there's a better term, we can use that instead and maybe get DS to change it on their end too.